.changelog/2024.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+plugin/k8s: Include deployment and release resources in `waypoint status` output
+```
```

Comment:
I'd add another improvement block for `plugin/k8s` that says the releaser status report is being updated for what its checked against too